sys/shell/commands/sc_netif.c

```diff
@@ -267,6 +303,40 @@ static void _netif_set_addr(kernel_pid_t dev, ng_netconf_opt_t opt,
     puts("");
 }
 
+static void _netif_set_state(kernel_pid_t dev, char *state_str)
+{
+    ng_netconf_state_t state;
+    if ((strcmp("off", state_str) == 0) || (strcmp("OFF", state_str) == 0)) {
+        state = NETCONF_STATE_OFF;
+    }
+    else if ((strcmp("sleep", state_str) == 0) ||
+             (strcmp("SLEEP", state_str) == 0)) {
+        state = NETCONF_STATE_SLEEP;
+    }
+    else if ((strcmp("idle", state_str) == 0) ||
+             (strcmp("IDLE", state_str) == 0)) {
+        state = NETCONF_STATE_IDLE;
+    }
+    else if ((strcmp("reset", state_str) == 0) ||
+             (strcmp("RESET", state_str) == 0)) {
+        state = NETCONF_STATE_RESET;
+    }
+    else {
+        puts("usage: ifconfig <if_id> set state [off|sleep|idle|reset]");
```

Comment:
(I mean in this function https://github.com/RIOT-OS/RIOT/pull/2710/files#diff-ad63ecb056f596bac854115d8ccb46ebR152) Alternatively write the states above lower-case.